src/Domain/CoordinateMaps/Frustum.hpp

```diff
@@ -30,13 +30,16 @@ namespace CoordinateMaps {
 /// (2.0,5.0), and with the upper base extending from (0.0,1.0) to (1.0,3.0),
 /// the corresponding value for `face_vertices` is `{{{{-2.0,3.0}}, {{2.0,5.0}},
 /// {{0.0,1.0}}, {{1.0,3.0}}}}`. The user may reorient the frustum by passing
-/// an `OrientationMap` to the constructor.
+/// an `OrientationMap` to the constructor. `with_equiangular_map` determines
```

Comment:
If `with_equiangular_map` is true, then apply a tangent function map to the logical...